### PR TITLE
Use assets directory from vagov-content

### PR DESCRIPTION
**Pull request by jbalboni**
## Description

This updates the build script to use assets from vagov-content, in addition to assets in vets-website.

Content directory PR is here: https://github.com/department-of-veterans-affairs/vagov-content/pull/61

## Testing done
Tested locally to make sure moved images still show up

## Acceptance criteria
- [ ] Assets from vagov-content show up
- [ ] Assets from vets-website also still show up correctly

## Definition of done
- [ ] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [ ] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

File: script/build.js [hunk 1]
```diff
@@ -56,7 +56,10 @@ smith.use(collections(BUILD_OPTIONS.collections));
 smith.use(leftRailNavResetLevels());
 smith.use(dateInFilename(true));
 
-smith.use(assets(BUILD_OPTIONS.assets));
+smith.use(assets(BUILD_OPTIONS.appAssets));
```
Comment (jbalboni) [inline script/build.js]: I went through a few different approaches here, but just calling the assets plugin twice with different source folders turned out to be the simplest way to make this work.

File: script/options.js [hunk 2]
```diff
@@ -115,6 +115,10 @@ function applyBrandConsolidationOverrides(options) {
     collections: require('./collections/brand-consolidation.json'),
     redirects: require('./vagovRedirects.json'),
     domainReplacements,
+    contentAssets: {
+      source: path.join(options['content-directory'], '../assets'),
+      destination: './',
+    },
```
Comment (ncksllvn) [inline script/options.js]: I don't think it hurts anything, but should this behind the WBC feature flag? `options['content-directory']` right now is only a WBC argument.

**Review by ncksllvn — APPROVED** [review]
LGTM!

File: script/options.js (hunk 2 above)
Comment (jbalboni) [reply script/options.js]: Yep, this is inside the `applyBrandConsolidationOverrides`

**Pull request merged**